public/css/synopsis.css

```diff
@@ -0,0 +1,23 @@
+body {
```

Comment:
well done having multiple css files, tha'ts a great way to structure when you work with larger sites.